src/services/utils/gql/types/parseNative.ts

```diff
@@ -31,10 +31,15 @@ export function inferTypeByTarget(target: Function, key?: string) {
   }
 
   const returnType = Reflect.getMetadata('design:returntype', target, key);
+
   if (returnType) {
     return returnType;
   }
 
+  const descriptor = Object.getOwnPropertyDescriptor(target, key);
```

Comment:
I think this function is already quite complicated and not obvious. Making external function like `isGetter` would make it more readable and obvious about what is the point here.